README.md

```diff
@@ -18,15 +18,20 @@ Available commands:
 /stop Stop the bot
 /set Set search attributes with this syntax: "\set hdd_count 3"
     Attributes are:
-        - `hdd_count`    int (min number of hard drives)
-        - `hdd_size`     int (min amount of GB for each disk)
-        - `raid`        enum ('raid5', 'raid6', 'None')
-        - `after_raid`  int (min size of raid after assembly in TB)
-        - `ram`         int (min RAM size in GB)
-        - `price`       int (max Price in Euro)
-        - `ecc`         bool (0 or 1)
-        - `inic`        bool (0 or 1)
-        - `hwr`         bool (0 or 1)
+        - `hdd_count`     int (min number of hard drives)
+        - `hdd_size`      int (min amount of GB for each disk)
+        - `raid`          enum ('raid5', 'raid6', 'None')
+        - `after_raid`    int (min size of raid after assembly in TB)
+        - `cores`         int (min number of cpu cores)
+        - `threads`       int (min number of cpu threads)
```

Comment:
I think we should skip the `threads` option, as this is basically always `cores *2` on modern CPUs.
No need to make it more complex

Reply:
it's not actually always cores * 2, because a bunch of new intel cpus use performance and efficiency cores and don't have multithreading on all of them. I'll just stick to using threads everywhere

Reply:
Uh. Even on the server CPUs? I thought those were only the low-energy laptop/mobile CPUs

Reply:
nope, it's on all cpus. there's already an example of such a cpu in the hetzner börse